### PR TITLE
golang: build.sh add misc directory

### PR DESCRIPTION
**Pull request by kcubeterm**
Add misc directory in golang root which contains useful wrapper for wasm, android etc

### CODE REVIEW

File: packages/golang/build.sh [hunk 1]
```diff
@@ -40,6 +40,7 @@ termux_step_make_install() {
 	cp pkg/include/* $TERMUX_GODIR/pkg/include/
 	cp -Rf lib/* $TERMUX_GODIR/lib
 	cp -Rf pkg/${TERMUX_GOLANG_DIRNAME}/* $TERMUX_GODIR/pkg/${TERMUX_GOLANG_DIRNAME}/
+        cp -Rf misc/ $TERMUX_GODIR/
```
Comment (Grimler91) [inline packages/golang/build.sh:43]: Please change the spaces to tab and bump the package

**Review by Grimler91 — CHANGES_REQUESTED** [review]

File: packages/golang/build.sh (hunk 1 above)
Comment (kcubeterm) [reply packages/golang/build.sh]: May I know, how u put tab in GitHub editor in android chrome

**Comment by soufainmqaiss** [comment]
Le ven. 4 sept. 2020 à 08:49, Henrik Grimler <notifications@github.com> a
écrit :

**Comment by Grimler91** [comment]
Thanks!

File: packages/golang/build.sh (hunk 1 above)
Comment (Grimler91) [reply packages/golang/build.sh]: No idea, but looks like you figured it out :)

**Pull request merged**